frontends/qt/main.cpp

```diff
@@ -163,6 +163,11 @@ int main(int argc, char *argv[])
     qputenv("DRAW_USE_LLVM", "0");
 #endif
 
+    // Force software rendering over GPU-accelerated rendering as various rendering artefact issues
+    // were observed on Windows and the app crashes on some Linux systems.
+    qputenv("QMLSCENE_DEVICE", "softwarecontext");
```

Comment:
I don't think you need this, esp. since we're upgraded to qt5.15 but I guess it doesn't do any harm.